### ISSUE TITLE
node-motoko causes syntax error

### ISSUE THREAD

**Issue by lastmjs** [demergent-labs/azle#749]
For some reason there is a syntax error when trying to incorporate node-motoko: https://github.com/dfinity/node-motoko.

Azle code like the following will create the error:

```typescript
import { Query, Update, nat64, blob } from 'azle';
import { Motoko } from 'motoko/versions/latest/moc.min.js';
// import Motoko from 'motoko';

let count: nat64 = 0n;

export function readCount(): Query<nat64> {
    return count;
}

export function incrementCount(): Update<nat64> {
    count = count + 1n;

    return count;
}

export function compile_motoko(): Query<blob> {
    const source =
        'actor { public query func example() : async Text { "Hello" } }';

    Motoko.saveFile('Main.mo', source);
    const result = Motoko.compileWasm('ic', 'Main.mo');

    console.log(result.code.wasm); // Uint8Array(...)

    return result;
}

```

Error:

```
expected token 'identifier', got 'of' in binding identifier at line 612, col 27628
```

**Comment by lastmjs** [comment]
Also Azle needs to use `r##"##` or some other sequence of `##`, because node-motoko has a `"#` in a case statement. I fixed this locally in Azle 0.7.0, but Azle 0.8.0 will need the fix as well since we changed that mechanism.

**Comment by lastmjs** [comment]
We probably also need the Wasm binary limit to be lifted in order to get this to work, considering the size of the  Motoko JS code is rather large
